Sources/AppStoreConnectCLI/Services/AppStoreConnectService.swift

```diff
@@ -62,6 +62,78 @@ class AppStoreConnectService {
         return BetaTester(output)
     }
 
+    func addTestersToGroup(
+        bundleId: String,
+        groupName: String,
+        emails: [String]
+    ) throws {
+        let testerIds = try emails.map {
+            try GetBetaTesterOperation(options: .init(id: nil, email: $0))
```

Comment:
`Options` are just an associated type, there's nothing really stopping us from using an enum for `GetBetaTesterOperation.Options`